.github/workflows/_hugo.yml

```
@@ -0,0 +1,55 @@
+name: Build Hugo Site
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      page_url: ${{ steps.some_step.outputs.page_url }}
+    steps:
+      - name: Install Hugo CLI
```

actionlint

"uses" is required to run action in step

(syntax-check)